src/mame/konami/ksys573.cpp

```diff
@@ -6092,7 +6150,7 @@ ROM_START( pcnfrk9ma )
 	ROM_LOAD( "gcd09aba.u6",   0x000000, 0x000008, BAD_DUMP CRC(ce84419e) SHA1(839e8ee080ecfc79021a06417d930e8b32dfc6a1) )
 
 	DISK_REGION( "cdrom0" )
-	DISK_IMAGE_READONLY( "d09aba02", 0, BAD_DUMP SHA1(e29c90652830041ec695adc43cbdd5023ae8ef67) )
+	DISK_IMAGE_READONLY( "d09aba02", 0, BAD_DUMP SHA1(a817d1c7fdb354b7d2d5c08f92a352c76a2b1a72) )
```

Comment:
I never received a replacement for this and can't remember why I changed it so I've reverted the change to d09aba02 (cc: @Tafoid).

Reply:
Noted.  Thank you for verifying and checking that out.